paasta_tools/utils.py

```diff
@@ -325,6 +326,12 @@ def validate_service_instance(service, instance, cluster, soa_dir):
         sys.exit(3)
 
 
+def compose(func_one, func_two):
```

Comment:
hmm, there's no explicit test for it but it was being covered because the tests do `--help` for each command. I'll add a test specifically for this though.